### DISCUSSION TITLE
Expression database for rat?

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#1069] [Questions & Problems]
Hello,

is there an expression database for other species than human? I can only find the human gene expression database under Open-Systems-Pharmacology/ Suite Downloads.

regards,
Carrie.

**Comment by tobiasK2001** [comment]
not yet!. See also here:
https://github.com/Open-Systems-Pharmacology/Forum/discussions/882

**Comment by StephanSchaller** [comment]
Hi @Carrie-nothing , currently, there have been activities on translating further databases to a PK-Sim compatible format.
This has been done for all PK-Sim relevant species for the database https://bgee.org/ and will soon be published.
